maintainers/maintainer-list.nix

```diff
@@ -11513,6 +11513,12 @@
       fingerprint = "A317 37B3 693C 921B 480C  C629 4A2A AAA3 82F8 294C";
     }];
   };
+  setthemfree = {
```

Comment:
Generally I'd suggest using your github name here, but whatever you prefer. People with try to do @ setthemfree and if it doesn't exist may give up rather than looking it up.

Reply:
Makes sense. Updated accordingly.